scripts/start-geth.sh

```
@@ -50,7 +50,7 @@ geth --datadir $DATA --password $pwdfile account import $tmpfile
 rm $tmpfile
 
 # start up
-geth --networkid 9000 --dev --dev.period 1 --datadir $DATA --http --http.addr localhost --http.api 'personal,eth,net,web3,txpool,miner,debug' \
+geth --dev --dev.period 1 --datadir $DATA --http --http.addr localhost --http.api 'personal,eth,net,web3,txpool,miner,debug' \
 -unlock '0x57f96e6b86cdefdb3d412547816a82e3e0ebf9d2' --password $pwdfile \
 --allow-insecure-unlock --ipcdisable
 $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)